src/Components/Profile/EditProfile.jsx

```diff
@@ -0,0 +1,76 @@
+import React, { Component } from "react";
+import toastr from "toastr";
+
+export class EditProfile extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			username: localStorage.getItem("username"),
+			bio: localStorage.getItem("bio"),
+			image: localStorage.getItem("image")
+		};
+		this.onChange = this.onChange.bind(this);
+		this.onSubmit = this.onSubmit.bind(this);
+	}
+	onChange(e) {
+		this.setState({ [e.target.name]: e.target.value });
+	}
+
+	onSubmit(e) {
+		e.preventDefault();
+		const profileData = {
+			username: this.state.username,
+			bio: this.state.bio,
+			image: this.state.image
+		};
+
+		fetch("http://localhost:8000/api/user/", {
+			method: "PUT",
+			body: JSON.stringify({ user: profileData }),
+			headers: {
+				"content-type": "application/json",
+				Authorization:
+					"TOKEN eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpZCI6MSwiZXhwIjoxNTQ5MDMxMTc2fQ.sl1pOnppzateccGlvoOiffxgdnYqseQVuPK9UGgJqTg"
+			}
+		})
+			.then(res => res.json())
+			.then(data => {
+				if (data) {
+					console.log(data);
```

Comment:
Could you remove the console log from the component

Reply:
@shakirandagire I removed the consoles